lib/main.dart

```diff
@@ -65,6 +65,7 @@ class Pangolin extends StatefulWidget {
   static OverlayState overlayState;
 
   static void setLocale(BuildContext context, Locale locale) {
+    EasyLocalization.of(context).locale = locale;
     _PangolinState state = context.findAncestorStateOfType<_PangolinState>();
```

Comment:
do we still need the call to pangolinstate here?

Reply:
Yes, otherwise the switcher in qs doesn't work

Reply:
Hive use its set value with settingsbox to persist locale string, 🤔   but I'd change it for get new strings on locale changing

Reply:
Yeah, we can basically try to move a bit away from hive. It already sets it inside the switch statement and calls it on the load so I think it can work without